core/app/models/spree/order.rb

```diff
@@ -154,7 +154,7 @@ def self.register_line_item_comparison_hook(hook)
 
     # For compatiblity with Calculator::PriceSack
     def amount
-      line_items.inject(0.0) { |sum, li| sum + li.amount }
+      line_items.map(&:amount).sum
```

Comment:
There is a difference here that needs to be noted. `.sum(:amount)` will do a round trip to the database doing the sum in SQL, where `.map(&:amount).sum` will use the values from the objects.

There are different circumstances where each will have higher performance depending on if we have the data loaded in to objects already or not.